index.html

```diff
@@ -165,19 +182,11 @@ <h2>Featured Mentors</h2>
                 </div>
                 <div class="col-md-8">
                   <div class="card-body">
-                    <h5 class="card-title">Bekjon Ishpulatov</h5>
-                    <p class="card-text">
-                      This is a wider card with supporting text below as a
-                      natural lead-in to additional content. This content is a
-                      little bit longer.
-                    </p>
+                    <h5 class="card-title"></h5>
+                    <p class="card-text description"></p>
                     <div class="line"></div>
                     <p class="card-text">
-                      <small class="text-muted"
```

Comment:
- [ ]  for the home page, Just as the previous reviewer mentioned, from the design guidelines It contains a top section with a column that includes a small line of text, the page title, a block of text, a medium line of text in bold letters, and a smaller line of text. I notice you take a special approach when it comes to background imaging and content positioning, This might be great in some cases. But as a front-end engineers, we need to simulate the provided design as much as possible. Please try to mimic the provided design before the next review request. 

### Give design
![219143551-5514207c-5cbf-42a9-9209-a363272148fc](https://user-images.githubusercontent.com/69077061/219751780-53ebd8b8-2754-4ce1-a5a9-e21a461e9422.png)

### Your design
![image](https://user-images.githubusercontent.com/69077061/219751979-43b9a5be-685d-464e-abef-d205863552ef.png)

Reply:
- [ ] As suggested, kindly implement it. Donot forget that the template is like a specification from your client which you don't really get to decide for yourself what to add or not to. This project is named at testing and strengthening you on certain `css` mastery, and I think you should enjoy the process :wink:  Apart from being a Software Developer, having good design instinct (User Interface and some User Experience gimmicks) is a nice-to-have treasure. :smile: 
_**In the section being required, did you see how if filled some empty spaces while yours remain too blank as if something is missing?**_ Please try to add something like the date when your training will commence.